central/plugins/org.jboss.tools.central/src/org/jboss/tools/central/editors/xpl/DiscoveryViewer.java

```diff
@@ -823,13 +823,15 @@ private void createEnvironment() {
 		if (bundle == null) {
 			bundle = Platform.getBundle("org.eclipse.mylyn.commons.core"); //$NON-NLS-1$
 		}
-		String versionString = (String) bundle.getHeaders().get("Bundle-Version"); //$NON-NLS-1$
-		if (versionString != null) {
-			Version version = new Version(versionString);
-			environment.put("org.eclipse.mylyn.version", version.toString()); //$NON-NLS-1$
-			environment.put("org.eclipse.mylyn.version.major", version.getMajor()); //$NON-NLS-1$
-			environment.put("org.eclipse.mylyn.version.minor", version.getMinor()); //$NON-NLS-1$
-			environment.put("org.eclipse.mylyn.version.micro", version.getMicro()); //$NON-NLS-1$
+		if (bundle != null) {
```

Comment:
Does that code makes any sense now that we're not using anymore Mylyn for discovery ?

Reply:
good point. looking where this environment can be used in JBossTools code base.